_about/release-cadence.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Build & Release Cadence
+description: How we manage, number, and support Istio releases.
+weight: 5
+---
+
+We produce new builds of Istio on a daily basis. Around once a month or so, we take one of these daily
+builds and run it through a number of additional qualification tests and tag the build as an Alpha release.
```

Comment:
I feel weird calling these alpha releases, since alpha has a specific meaning in terms of features, and this is unrelated to that. Can we just call them monthlies?  Prerelease? Anything other than alpha/beta/stable would be useful here.

Reply:
Oh, yeah, the use of alpha here will be confusing with what we use for individual features. I'll change to 'prerelease'.

Reply:
standard is "rc" for the short form in the semver suffix